content/en/docs/contribute/intermediate.md

```diff
@@ -513,6 +532,51 @@ most up-to-date version of that branch.
       the branch to your fork, and the pull request should no longer show any
       conflicts.
 
+12. If you have multiple commits on your PR, you have to squash them into a
+    single commit before you PR can be merged. You can check the number of commits
+    on your PR's `Commits` tab or by running `git log` locally. Squashing commits is a form of rebasing.
+
+    ```bash
+    git rebase -i HEAD<number_of_commits>
+    ```  
+
+    The `-i` switch tells git you want to rebase interactively, which enables
+    you to tell git which commits you need to squash into the first one. For
+    example, let's say you have 3 commits on your branch:
+
+    ```bash
+    12345 commit 4 (2 minutes ago)
+    6789d commit 3 (30 minutes ago)
+    456df commit 2 (1 day ago)     
+    ```
+
+    You need to squash your last three commits into the first one.
+
+    ```bash
+    git rebase -i HEAD~3
+    ```
+
+    That command will open up an editor with the following:
```

Comment:
```suggestion
    That command opens an editor with the following:
```

Reply:
GAAAH! Got me on the style guide AGAIN!  I plead illness as my defense, and will now triple check my own work before pushing.